File: 1316/1316.py
num = int(input())
cnt = 0

for i in range (num):
	word = input()
	if (list(word) == sorted(word, key=word.find)): #알파벳을 찾은 순으로 전부 배열한 리스트와 같아야 그룹단어.
		cnt += 1
print(cnt)

File: 1316/1316.cpp
#include <iostream>
#include <string>
using namespace std;

int main(void)
{
	int n;
	int k;
	int cnt = 0;
	bool	flag = true;
	string word;
	int temp[26];

	cin >> n;
	for(int i = 0; i < n; i++)
	{
		cin >> word;
		k = 0;
		while(k < 26)
			temp[k++] = 0;
		flag = true;
		for (int j = 0; j < word.size() && flag == true; j++)
		{
			if (temp[word[j] - 'a'] == 0)
			{
				temp[word[j] - 'a'] = 1;
				flag = true;
			}
			else if (temp[word[j] - 'a'] == 1 && word[j-1] != word[j])
				flag = false;
		}
		if (flag == true)
			cnt++;
	}
	cout << cnt;
	return (0);
}